ntfysh_client/SettingsDialog.cs

```diff
@@ -37,5 +73,20 @@ private void cancelButton_Click(object sender, EventArgs e)
         {
             DialogResult = DialogResult.Cancel;
         }
+
+        private void setNotificationsUIElements()
+        {
+            groupCustomNotificationSettings.Enabled = useCustomTrayNotifications.Checked;
+            timeoutLabel.Text = useCustomTrayNotifications.Checked ? _customNotificationsTimeout : _windowsNotificationsTimeout;
+            timeout.Minimum = useCustomTrayNotifications.Checked ? -1 : 0;
+        }
+
+        private void useCustomTrayNotifications_CheckedChanged(object sender, EventArgs e)
+        {
+            setNotificationsUIElements();
+        }
+
+        private const string _windowsNotificationsTimeout = "Notification Toast Timeout (seconds, may be ignored by OS based on accessibility settings):";
+        private const string _customNotificationsTimeout = "Notification Toast Timeout (seconds, use -1 to require closing notification):";
```

Comment:
Would it be better to make `0` no timeout requiring closing notifications instead of `-1`? My logic behind this is `0` is not a valid value anyway, and would result in the immediate closure of the notification, which seems somewhat pointless and a bug. Then we don't end up with an invalid value between two valid values (`-1` and `1`) and can enforce a minimum value of `0` easily.

Reply:
0 for not timeout makes sense. Although, a 0 was valid before.

(but...., If I understand correctly, a default Windows install will ignore anything outside of 1-10 and just use 10 instead, so a 0 before became 10 s, ...., but the way we have it setup now anyone upgrading will have to open the settings dialogue to switch over anyays...)

I'm leaning toward using 0.

Reply:
Changed to 0 and dropped -1 handling/updating